Monika After Story/game/script-topics.rpy

```diff
@@ -38,7 +38,24 @@ init 11 python:
 
 #BEGIN ORIGINAL TOPICS
 
+
 #Use this topic as an example
+init 5 python:
+    # List of keywords for the topic.
+    for key in ['film''movie']:
```

Comment:
items in `python` lists are separated by commas ( , ). Please separate the strings in this list with a comma.